airbyte-integrations/connectors/source-hubspot/source_hubspot/manifest.yaml

```diff
@@ -508,13 +508,56 @@ definitions:
         class_name: source_hubspot.components.MigrateEmptyStringState
         cursor_field: updatedAt
 
+  deal_pipelines_stream:
+    $ref: "#/definitions/stream_base"
+    name: deal_pipelines
+    primary_key:
+      - pipelineId
+    retriever:
+      $ref: "#/definitions/base_retriever"
```

Comment:
something interesting i noticed about the `/crm-pipelines/v1/pipelines/deals` endpoint is that I don't think pagination is actually used or supported. So I think to make this stream more accurate to what it's doing we shouldn't use the `base_retriever and just define `type: SimpleRetriever` and not define a `paginator` component.

docs:
https://developers.hubspot.com/docs/reference/api/crm/pipelines/v1